nikola/plugins/task/pages.py

```diff
@@ -43,9 +45,17 @@ def gen_tasks(self):
             "filters": self.site.config["FILTERS"],
             "show_untranslated_posts": self.site.config['SHOW_UNTRANSLATED_POSTS'],
             "demote_headers": self.site.config['DEMOTE_HEADERS'],
+            "disable_indexes": self.site.config["DISABLE_INDEXES_PLUGIN_INDEX_AND_ATOM_FEED"],
```

Comment:
Is it a good idea to put this into `kw`? After all, this value is only needed at task generation time, and its use does not require a rebuild of all pages if it changes.

Reply:
Sure, it’s not really needed.